### PR TITLE
markdown numbered headers

### PR DESCRIPTION
**Pull request by weituotian**
repository：[md_numbered_headers](https://github.com/weituotian/md_numbered_headers)
[tag 1.0.1](https://github.com/weituotian/md_numbered_headers/releases/tag/1.0.1)

### CODE REVIEW

**Comment by AlexKvazos** [comment]
Breaks on usage 🐞
screen shot 2017-05-11 at 2 06 45 pm

**Comment by weituotian** [comment]
@AlexKvazos i fix it with [tag 1.02](https://github.com/weituotian/md_numbered_headers/tree/1.0.2)

File: repository/m.json [hunk 1]
```diff
@@ -355,6 +355,17 @@
 			]
 		},
 		{
+			"name": "Markdown Numbered Headers",
+			"details": "https://github.com/weituotian/md_numbered_headers",
+			"labels": ["markdown", "headers", "numbered headers"],
+			"releases": [
+				{
+					"sublime_text": ">=3000",
+					"branch": "master"
```
Comment (wbond) [inline repository/m.json]: We don't support branch-based releases for new packages. You need to change this line to `"tags": true`.

Comment (weituotian) [inline repository/m.json]: I had replaced "branch": "master" with "tags": true
and what more i should do?

**Pull request merged**